src/main/openapi/ds-image-openapi_v1.yaml

```diff
@@ -462,8 +392,150 @@ paths:
+              application/json:
+                schema:
+                  type: object
+                  properties:
+                    TileSize:
+                      type: integer
+                    Overlap:
+                      type: integer
+                    Format:
+                      type: integer
+                    Size:
+                      type: object
+                      properties:
+                        Width:
+                          type: integer
+                        Height:
+                          type: integer
+                example:
+                - TileSize: 256
+                  Overlap: 0
+                  Format: "jpg"
+                  Size:
+                    Width: 3505
+                    Height: 4752
+              application/xml:
+                schema:
+                  xml: 
+                    name: 'Image'
+                    namespace: 'http://schemas.microsoft.com/deepzoom/2008'
+                  type: object
+                  properties:
+                    TileSize:
+                      type: integer
+                      xml:
+                        attribute: true
+                      example: 256
+                    Overlap:
+                      type: integer
+                      xml:
+                        attribute: true
+                      example: 0
+                    Format:
+                      type: integer
+                      xml:
+                        attribute: true
+                      example: 'jpg'
+                    Size:
+                      type: object
+                      properties:
+                        Width:
+                          type: integer
+                          xml:
+                            attribute: true
+                          example: 3505
+                        Height:
+                          type: integer
+                          xml:
+                            attribute: true
+                          example: 4752
                   
- # The ping service should be in all projects, should not do any advanced processing
+  /deepzoom/{imageid}_files/{layer}/{tiles}.{format}:
+    get:
+      tags:
+        - 'Access'
+      summary: 'DeepZoom Tile'
+      description: |- 
+        DeepZoom can bee used with the Internet Imaging Protocol (IIP). This endpoint only requires the DeepZoom parameter to work. Besides, this endpoint has the capability to make use of the IIP parameters shown below. 
+      operationId: getDeepzoomTile
+      parameters:
+      - name: imageid
+        in: path
+        description: 'Identifier/path for image.'
+        required: true
+        schema:
+          type: string
+        example: "image/path.tif"
+      - name: layer
+        in: path
+        description: 'Zoom layer for the tile'
+        required: true
+        schema:
+          type: integer
+      - name: tiles
+        in: path
+        description: 'Tile number'
```

Comment:
Maybe add something like "Tile specified as `x_y` at the given layer? Or is that too much information?